Glycine/max/wip_marlene/Pan_Yu_2021

```diff
@@ -0,0 +1,20 @@
+gene_symbols:
+  - GmABCC8
+gene_symbol_long: Glycine max ATP-binding cassette transporter
+gene_model_pub_name: Glyma.07G011600.1
+gene_model_full_id: glyma.Wm82.gnm2.ann1.Glyma.07G011600 
```

Comment:
The gene ID here is for Glyma.07G011600, not Glyma.07G011600.1.  It sounds like the right gene from the description.  Are these the same?